test_code_for_qnv.py

```diff
@@ -0,0 +1,168 @@
+import numpy as np
+import qnv
+
+A=np.array([1,2,3])
+B=np.array([-1,-2,-3])
+C=np.array([1,-2,-3])
+D=np.array([4,-6,7])
+E=np.array([0,-3,2])
+F=np.array([0,0,0])
+
+flag=1 #to check whether code passes all test cases for dot. flag will be made 0 if it fails any test case
+
+G=qnv.dot(A,A)
+if G != 14:
+	flag=0
+
+G=qnv.dot(B,B)
+if G != 14:
+	flag=0
+
+G=qnv.dot(A,B)
+if G != -14:
+	flag=0
+
+G=qnv.dot(C,D)
+if G != -5:
+	flag=0
+
+G=qnv.dot(E,A)
+if G != 0:
+	flag=0
+
+G=qnv.dot(F,E)
+if G != 0:
+	flag=0
+
+G=qnv.dot(F,F)
+if G != 0:
+	flag=0
+
+if flag == 1:
+	print ("all cases passed for dot")
+else:
+	print ("error for dot")
+
+flag=1 #to check for cross
+cross=np.array([0,0,0])

```

Comment:
It's better if you don't have the name for variable same as the function. change it to `expected_cross` etc